Documentation/LibraryExtensibleRegexLiterals.md

```diff
@@ -1,8 +1,37 @@
-# Library-Extensible Regex Literals
+# Regex Literals Musings
 
 * Author: Michael Ilseman
 
-## Example
+## Introduction
+
+As forums discussions bear fruit, I wanted to write down some of my emerging thoughts and musings on the topic of regex literals.
+
+Main points:
+
+- Go with a typical regex literal instead of something custom/nicer
+    + Main reason for regex is familiarity and broad appeal
+    + If we're building something custom, let's not do it on top of the shaky technical foundation that is regex
```

Comment:
I'm not sure I'd describe regular expressions as having a "shaky technical foundation", FWIW. Sure, the common backtracking implementations, possibly, as they've added all kinds of ad-hoc extensions, but regular expressions _per se_ are a compact notation for regular grammars and have a very solid foundation in formal language theory.